crates/matrix-sdk-base/src/rooms/normal.rs

```diff
@@ -984,6 +998,41 @@ impl RoomInfo {
     }
 }
 
+#[cfg(feature = "experimental-sliding-sync")]
+fn apply_redaction(
+    event: &Raw<AnySyncTimelineEvent>,
+    raw_redaction: &Raw<OriginalSyncRoomRedactionEvent>,
+    room_version: &RoomVersionId,
+) -> Option<Raw<AnySyncTimelineEvent>> {
+    use ruma::canonical_json::redact_in_place;
+
+    let mut event_json = match event.deserialize_as() {
```

Comment:
uber-nit: here and below, could use `let else` syntax

Reply:
I would love it if you were right but unfortunately, `let Ok(…) else` doesn't let you access the error in the else branch.